slacktest/handlers.go

```diff
@@ -26,7 +26,12 @@ func contextHandler(server *Server, next http.HandlerFunc) http.Handler {
 	})
 }
 
-func usersInfoHandler(w http.ResponseWriter, r *http.Request) {
+// handle auth.test
+func authTestHandler(w http.ResponseWriter, _ *http.Request) {
```

Comment:
The `r` variables were unused in some of the handler methods, so I changed them to the blank identifier.